### ISSUE TITLE
rcon levels privilages all the same level 1 to 3

### ISSUE THREAD

**Issue by alandball** [AdVetter/XXLDDRace#6]
level_command changes privileges for levels 1, 2, 3 which is probably not the wanted behaviour.

Change back level_command to mod_command, add subadmin_command, helper_command, subadmin_status, helper_status, so that each level is independently configured. This would seem like the wanted behaviour.

**Comment by AdVetter** [comment]
I thought about this, but it does not make much sense, if the subadmin has less rights then a mod.

**Comment by alandball** [comment]
No, I was being a noob. To me 0 was off and 1 on, but there are several options, as I discovered by entering 2, 3, and 4 which changes the permissions. I wasn't trying to say I want subadmin to have less rights than a mod. I put level_command ban 1 and saw all levels could then ban. I didnt see a way of only allowing subadmin and mod banning.

But looking at source late last night I saw how to correct my problem. Put the number of the level you want to enable the command to go down to, so 1 would be all from 4 down to 1 etc. Thanks, is nice feature.
